### PR TITLE
Update changelog for v0.38.1 release

### PR DESCRIPTION
**Pull request by rmfitzpatrick**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +55,7 @@ This Splunk OpenTelemetry Connector release includes changes from the [opentelem
 
 - [`httpsink` exporter](https://github.com/signalfx/splunk-otel-collector/tree/main/internal/exporter/httpsinkexporter) to make span data available via a HTTP endpoint
 - Initial release of [`translatesfx`](https://github.com/signalfx/splunk-otel-collector/tree/main/cmd/translatesfx) to translate a SignalFx Smart Agent configuration file into a configuration that can be used by an OpenTelemetry Collector
+- Add Chocolately package support. (#724)
```
Comment (jcheng-splunk) [inline CHANGELOG.md:58]: This is in main but hasn't actually been released yet.

**Review by mstumpfx — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,12 @@
 
 ## Unreleased
 
-- Add Chocolately package support. (#724)
+## v0.38.1
+
+### 🧰 Bug fixes 🧰
+
+- Fix evaluating env variables in ecs ec2 config (#930)
+- Correct certifi CA bundle removal from Smart Agent bundle (#933)
```
Comment (mstumpfx) [inline CHANGELOG.md:10]:
```suggestion
- Correct certificate CA bundle removal from Smart Agent bundle (#933)
```

Comment (rmfitzpatrick) [reply CHANGELOG.md:10]: this is the python library name

File: CHANGELOG.md (hunk 1 above)
Comment (rmfitzpatrick) [reply CHANGELOG.md:58]: oh, I assumed it was an oversight

**Pull request merged**